autoPyTorch/api/base_task.py

```diff
@@ -1506,7 +1520,7 @@ def predict(
 
         predictions = self.ensemble_.predict(all_predictions)
 
-        self._clean_logger()
+        self._cleanup()
```

Comment:
by the time that the code reaches fit pipeline. The Dask infrastructure is usually already closed. So, trying to close it would make no difference as we are checking if it is open or not. Also, this function does not need the dask infrastructure so if it is called in isolation we would not have the dask server open. I think it's safer to always close the servers when we are exiting a function just to be safe. I dont think we would run into any issues because of this

Reply:
Okay, that would not hurt I guess. As long as there is no error being thrown if it is already closed.